app/src/main/java/com/cornellappdev/score/components/GameScoreHeader.kt

```diff
@@ -0,0 +1,105 @@
+package com.cornellappdev.score.components
+
+import android.app.GameState
+import android.graphics.Paint.Align
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.score.R
+import com.cornellappdev.score.theme.Style.scoreHeaderText
+import com.cornellappdev.score.theme.Style.vsText
+
+@Composable
+fun GameScoreHeader(
+    leftTeamLogo: Painter,
+    rightTeamLogo: Painter,
+    gradientColor1: Color,
+    gradientColor2: Color,
+    modifier: Modifier
+) {
```

Comment:
Ooh the gradient looks really nice here, great stuff!

Reply:
Really like the use of `modifier` input too! Oh but one suggestion, `modifier` should always be optional as `modifier: Modifier = Modifier`; this supplies the default `Modifier`.